pkg/oc/bootstrap/docker/openshift/helper.go

```diff
@@ -895,11 +898,9 @@ func (h *Helper) updateConfig(configDir string, opt *StartOptions) error {
 			return err
 		}
 		glog.V(5).Infof("cgroup driver from Docker: %s", cgroupDriver)
-		if nodeCfg.KubeletArguments == nil {
-			nodeCfg.KubeletArguments = configapi.ExtendedArguments{}
-		}
 		nodeCfg.KubeletArguments["cgroup-driver"] = []string{cgroupDriver}
 	}
+	nodeCfg.KubeletArguments["fail-swap-on"] = []string{"false"}
```

Comment:
this appears to have broken backwards compatibility for cluster up.

Reply:
```
 error: Invalid NodeConfig /var/lib/origin/openshift.local.config/node-localhost/node-config.yaml
       flag: Invalid value: "fail-swap-on": is not a valid flag
```

Reply:
will update with a version gated check

Reply:
sadly it's going to have to be something like "greater than 3.8.0-alpha.0".

Reply:
oh, I know. and I'm sad

Reply:
https://github.com/openshift/origin/pull/17410